website_apps_store/controllers/main.py

```diff
@@ -67,26 +69,60 @@ def _get_search_domain(
             if attrib:
                 domain += [("attribute_line_ids.value_ids", "in", ids)]
 
+        if post.get("version"):
+            field_name = (
+                "product_variant_ids.product_template_attribute_value_ids"
+                + ".product_attribute_value_id.id"
+            )
+            domain += [
```

Comment:
Details but:

- using `osv.expression.AND() is better in terms of readability
- is calling twice `post.get(...)` enhanced by adding variables ?

Reply:
thanks for review !

Yeah I was just moving code ! you are right, will improve that :)